modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/search/FaroProjectIndexer.java

```diff
@@ -273,6 +284,16 @@ protected void doReindex(String[] ids) throws Exception {
 		indexableActionableDynamicQuery.performActions();
 	}
 
+	private double _getUsage(long count, long limit) {
+		if ((count == 0) || (limit == 0)) {
+			return 0.0;
+		}
+
+		double usage = 100.0 * count / limit;
+
+		return (long)(usage * 100) / 100.0;
```

Comment:
@CaioERP, can you explain the reason of you are casting to long if the method expects you return a double?